exercises/bowling/bowling_test.go

```diff
@@ -0,0 +1,68 @@
+package bowling
+
+import "testing"
+
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
+	}
+}
+
+func applyPreviousRolls(g *Game, rolls []int) error {
+	for _, pins := range rolls {
+		if err := g.Roll(pins); err != nil {
+			return err
```

Comment:
It could be possible to have a more helpful error message when `applyPreviousRolls` errors if this would return:

* the index of the erroring roll
* the number of pins in the erroring roll

I think that would help a student debug if there is a problem applying previous rolls.